app/models/actions/base_action.rb

```diff
@@ -0,0 +1,24 @@
+module Actions
```

Comment:
My proposal is to move all the Actions from Models namespace one level up, to their own namespace, or to the Interactions namespace to Interactions/Actions.

As far as I can see Actions have nothing to do with Models and data storage pattern, they are more of service object or interaction pattern. So from architectural point of view atm we got Interactions namespace dedicated to some atomic actions... and introducing /Model/Actions seems a bit weird to me. I know, I know, that's a legacy practice — so why shouldn't we introduce the new way of actions and sometimes in the future use just these actions to make entity changes?

Reply:
I think this conversion might be a good idea, thank you. As we had some previously made actions at that path I automatically continued without any hesitation.